embassy-net-esp-hosted/src/control.rs

```diff
@@ -120,7 +120,7 @@ impl<'a> Control<'a> {
             pwd: unwrap!(String::try_from(password)),
             bssid: String::new(),
             listen_interval: 3,
-            is_wpa3_supported: false,
+            is_wpa3_supported: true,
```

Comment:
This commit message should be as TODO in code..

Reply:
what's todo? WPA3 works flawlessly now.

Reply:
Commit came with following message:
> I've noticed wpa3 still works without this flag, so I'm not sure what this does tbh...

Reply:
if WPA3 is working well, the chances of that TODO ever getting looked at are basically zero... I don't think adding it is worth it.